### PR TITLE
Removed abberviation in namespace and added preamble for SRO to report conditions early

### PR DESCRIPTION
**Pull request by zvonkok**
Removed abberviation in namespace and added preamble for SRO to report conditions early

### CODE REVIEW

File: config/default/kustomization.yaml [hunk 1]
```diff
@@ -41,4 +41,5 @@ resources:
   - ../manager
   - ../prometheus
   - ../clusteroperator
+  - ../cr
 namespace: openshift-sro
```
Comment (ArangoGutierrez) [inline config/default/kustomization.yaml:45]: Shouldn't this ns also be changed?

**Review by ArangoGutierrez — COMMENTED** [review]
Looks good, just a minor comment

File: config/default/kustomization.yaml (hunk 1 above)
Comment (zvonkok) [reply config/default/kustomization.yaml:45]: This is a runtime settings, changed by kustomize when deployed via make deploy. Kustomize will read the setting from the Makfile. 

For CVO the important manifests are in /manifests

Comment (ArangoGutierrez) [reply config/default/kustomization.yaml:45]: Yeah that is what I was thinking, but wanted to ask, I still don't fully understand Kustomize yet 
/lgtm

**Comment by dagrayvid** [comment]
Overall it looks good, but can you explain (maybe in a comment in the CR manifest) the reason for having the preamble be a specialresource?

**Comment by zvonkok** [comment]
@dagrayvid This is to kickoff the reconcile loop. Only in the reconcile loop we have all tools to update cluster resources. All the other execution is happening in some vendor libraries.

**Comment by dagrayvid** [comment]
Thanks for the response.

/lgtm

**Pull request merged**